src/types.ts

```diff
@@ -0,0 +1,24 @@
+import { CumulusMessage } from '@cumulus/types/message';
+import { Context } from 'aws-lambda';
+import { ChildProcessWithoutNullStreams } from 'child_process';
+
+export type cumulusMessageAdapterError = { stderrBuffer: string };
```

Comment:
Do we want to make these interfaces rather than types?

Reply:
I'm a bit torn on these, as the intent of many of them is to be used internal to the package.

Also this type/interface should be CumulusMessageAdapterError given the it's an interface to a class.